components/scream/src/control/atmosphere_driver.cpp

```diff
@@ -312,6 +313,8 @@ void AtmosphereDriver::setup_surface_coupling_processes () const
     EKAT_ASSERT_MSG(exporter_found, "Error! SurfaceCoupling exporter data was setup, but no atm process "
```

Comment:
One thing I noticed recently is that EKAT_ASSERT_MSG (disabled ifdef NDEBUG) is being used where EKAT_REQUIRE_MSG (never disabled) is probably intended.